runtime/power.S

```diff
@@ -288,6 +288,15 @@ ENDFUNCTION caml_call_gc
 
 /* Call a C function from OCaml.  Function to call is in C_CALL_FUN */
 
+.macro RET_FROM_C_CALL
+        lbz     TMP, Caml_state(action_pending)
+        cmplwi  TMP, 0
+        beqlr   0
```

Comment:
Bonus point for using `beqlr` (conditional branch to the link register), but we can mark this branch as (very) likely taken, it might help some POWER processors with the branch prediction.
```suggestion
        beqlr+  0
```